test/test_mp_logger.py

```diff
@@ -176,7 +176,9 @@ def test_child_process_logging(self, tmpdir):
         child_process = Process(target=child_proc_logging_exception())
         child_process.daemon = True
         child_process.start()
-        openwpm_logger.close()
+        time.sleep(2)
         child_process.join()
+        print("Child processes joined...")
```

Comment:
This seems to fix this test. Could you please remove the `skipif` so it runs in CI again.

Reply:
I don't think this was me? But I removed the skipif

Reply:
Oh then that must have been me ^^ Thanks anyways